### PR TITLE
Fix tests to avoid OOMs

### PR DESCRIPTION
**Pull request by nileema**

### CODE REVIEW

File: presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java [hunk 1]
```diff
@@ -571,14 +571,14 @@ public void testMaxMinStringWithNulls()
     public void testGroupByPartitioningColumn()
             throws Exception
     {
-        assertQuery("SELECT orderkey, count(*) FROM lineitem GROUP BY orderkey");
+        assertQuery("SELECT orderkey, count(*) FROM lineitem WHERE orderkey % 8 = 0 GROUP BY orderkey");
```
Comment (martint) [inline presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java]: This one shouldn't matter. I think it's the test below that's causing problems

File: presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java [hunk 2]
```diff
@@ -571,14 +571,14 @@ public void testMaxMinStringWithNulls()
...
     }
 
     @Test
     public void testJoinPartitionedTable()
             throws Exception
     {
-        assertQuery("SELECT * FROM lineitem a JOIN lineitem b ON a.orderkey = b.orderkey");
+        assertQuery("SELECT * FROM lineitem a JOIN lineitem b ON a.orderkey = b.orderkey WHERE a.orderkey % 8 = 0 ");
```
Comment (martint) [inline presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java]: can we join orders vs lineitem, instead?

@haozhun, what was the change you made trying to achieve?

**Comment by haozhun** [comment]
We can remove these tests for now because partitioned table or partitioning column are no longer valid concepts in the engine. I'll add them to separate tests when needed.

**Comment by haozhun** [comment]
Looks good

**Pull request closed without merging**